README.md

```diff
@@ -88,7 +86,10 @@ When deploying the multi cluster traffic controller using the make targets the f
 ```sh
 export KUBECONFIG=./tmp/kubeconfigs/mctc-workload-1.kubeconfig
 ```
-1. Update the secret in `config/agent/secret.yaml` with the correct credentials for the control plane `tmp/kubeconfigs/mctc-control-plane.kubeconfig` . (**N.B.** The server should be changed to the kind cluster address e.g. `https://127.0.0.1:46419`)
+1. Update the secret in `config/agent/secret.yaml` with the correct credentials for the control plane
+```sh
+make agent-secret
+```
```

Comment:
This probably won't work in all cases. The host needs to change depending on where you are running the controller. If the controller is deployed, then this will work, but if it's running locally, you need to use the non internal one e.g. https://127.0.0.1:46419.

We should at least leave the comment here explaining that.

Reply:
what I did was create both secrets (mctc-workload-1 and mctc-workload-1-external) then you pass in which one to use as an argument to the agent (external for deployment and regular for local dev)

Reply:
though actually, I'd prefer to merge this now, and then I can open the spec syncing part of the dumb-agent on top of this, and it can be tweaked  after.

Reply:
I added a commit to allow passing the `LOCAL_ACCESS` environment variable so, if it's set, it uses the `127.0.0.1` address (doesn't add the `--internal` flag), and updated the README to include it when running the agent locally